src/Widgets/Block/DisplayTranslation.php

```diff
@@ -19,6 +19,12 @@ public function add_hooks() {
 		     ->then( spreadArgs( function ( $content ) use ( $getStringsFromMOFile ) {
 			     $strings = $getStringsFromMOFile( \get_locale() );
 
+			     $imageTranslator = make( \WPML_Media_Translated_Images_Update::class );
```

Comment:
@strategio I think this should be moved to Media plugin and Media plugin should hook `widget_block_content` before this.

Reply:
@brucepearson We discussed it with Pierre before when we were working on the first Gutenberg task. I also initially put my code into ST, but Pierre explained that it is better to keep it here. 

1. Now, the whole Gutenberg related code is stored in one place
2. if something changes just in Gutenberg, the compatibility is able to release a fix without WPML

Reply:
ok

Reply:
I have been hesitating, both ways have pros and cons...

In the end, I think it's more correct to move the media filtering to WPML Media.